### PR TITLE
add SJET-AOCODAF722MINIV2

### PR DESCRIPTION
**Pull request by aocodarc**
Upload file f7miniv2

### CODE REVIEW

File: configs/default/SJET-AOCODAF722MINIV2.config [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+set dshot_burst = ON
+set dshot_bitbang = AUTO
+set dshot_bitbang_timer = AUTO
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 400
+set beeper_inversion = ON
+set beeper_od = OFF
+set beeper_frequency = 0
+set max7456_clock = NOMINAL
```
Comment (haslinghuis) [inline configs/default/SJET-AOCODAF722MINIV2.config:132]: Please remove default values:

```suggestion
```

File: configs/default/SJET-AOCODAF722MINIV2.config [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+set max7456_spi_bus = 2
+set max7456_preinit_opu = OFF
+set led_inversion = 0
+set usb_msc_pin_pullup = ON
```
Comment (haslinghuis) [inline configs/default/SJET-AOCODAF722MINIV2.config:136]: Please remove default values:
```suggestion
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by aocodarc** [comment]
I have deleted the default configuration

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

File: configs/default/SJET-AOCODAF722MINIV2.config [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+timer B05 AF2
+# pin B05: TIM3 CH2 (AF2)
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer C08 AF3
+# pin B03: TIM8 CH3 (AF1)
+timer C09 AF3
+# pin A15: TIM8 CH4 (AF1)
```
Comment (nerdCopter) [inline configs/default/SJET-AOCODAF722MINIV2.config:79]: @aocodarc , these comments and pins do not match.

**Pull request merged**